api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -79,6 +79,15 @@ def format_deposit_type(deposit_type: finance_models.DepositType) -> str:
             return "Aucune information"
 
 
+def format_active_deposit(booking: bookings_models.Booking) -> str:
+    if booking.deposit:
+        if booking.deposit.expirationDate and booking.deposit.expirationDate > datetime.datetime.utcnow():
+            return Markup(
+                '<span class="mx-2 pb-1 badge rounded-pill text-bg-success"><i class="bi bi-check-circle"></i></span>'
+            )
+    return Markup('<span class="mx-2 pb-1 badge rounded-pill text-bg-dark"><i class="bi bi-x-circle"></i></span>')
```

Comment:
Je ne comprends pas pourquoi un badge avec juste une icône.
On a le genre d'affichage demandé dans le ticket pour indiquer par exemple si un lieu est permanent ou pas, dans l'onglet « Lieux associés » d'une structure. J'imaginais donc que tu allais faire quelque chose de similaire, pour une UI cohérente : 
![image](https://github.com/pass-culture/pass-culture-main/assets/23212130/248b4738-461e-45aa-af06-76c5ee7272da)
Éventuellement avec la couleur (le ticket ne précise pas si c'est souhaité ou non), mais pas dans un badge, on n'a pas ce genre d'affichage dans le BO, donc restons homogène.
Je base ma revue sur la description du ticket et l'homogénéité de l'application, mais demande si besoin des précisions de la demande produit (PM).